terraform/container-registry.tf

```diff
@@ -0,0 +1,64 @@
+resource "aws_ecs_cluster" "data_refinery" {
+  name = "data-refinery"
+}
+
+resource "aws_iam_role" "ecs_service_role" {
+  name = "data-refinery-ecs-service"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
```

Comment:
The date is coming from AWS' documentation where it says "Copy this text". I bet it's not truly necessary, but the documentation has it and it works so I went with it. I'm adding links to the sources of the policies in comments.